modules/configuration/test/ConfigurationTest.php

```diff
@@ -150,7 +150,7 @@ public function testAllLinks()
         $this->_linkTest("WWW");
         $this->_linkTest("Dashboard");
         $this->_linkTest("Imaging Modules");
-        $this->_linkTest("Statistics");
+        // $this->_linkTest("Statistics"); will be rewritten by REACT test
```

Comment:
Is upgrading Selenium making this test fail?

Reply:
Yes, it makes this test failed.

Reply:
Why this one and not the others?

Reply:
I am not sure, it shows an exception of a null element. 
It will be rewritten in this PR https://github.com/aces/Loris/pull/4939.